### PR TITLE
Feature/sprint ioda converters nick e

### PR DESCRIPTION
**Pull request by nicholasesposito**
As part of the mini code sprint on 1/26/23, naming conventions changes were made in a number of files in GDASApp. The focus of this PR was for aircraft data in the aircraft.yaml in the testing directory to be updated. There were a few small changes to run_ufo_hofx_test.sh and the eva file jedi_gsi_compare_conv.yaml.

### CODE REVIEW

**Review by ADCollard — CHANGES_REQUESTED** [review]
Please remove ush/ufoeval/run_ufo_hofx_test.sh from this PR.

YAML fails benchmark test (I assume this is expected).  We will address after merge to develop.

File: ush/eva/jedi_gsi_compare_conv.yaml [hunk 1]
```diff
@@ -349,7 +349,7 @@ diagnostics:
             x:
               variable: experiment::ErrDiff::${variable}
             y:
-              variable: experiment::MetaData::air_pressure
+              variable: experiment::MetaData::pressure
```
Comment (CoryMartin-NOAA) [inline ush/eva/jedi_gsi_compare_conv.yaml:352]: @PraveenKumar-NOAA this will need to be changed in your version of this file until it's merged in

Comment (PraveenKumar-NOAA) [reply ush/eva/jedi_gsi_compare_conv.yaml:352]: @CoryMartin-NOAA thanks, it worked.

**Comment by nicholasesposito** [comment]
> Please remove ush/ufoeval/run_ufo_hofx_test.sh from this PR.
> 
> YAML fails benchmark test (I assume this is expected). We will address after merge to develop.

@ADCollard Done!

**Review by ADCollard — APPROVED** [review]

**Pull request merged**